api/src/main/resources/messages.properties

```diff
@@ -2177,6 +2177,7 @@ User.creating.already.exists=The user {0} already exists, it could therefore not
 User.setPersonId=You need to call setPerson(Person)
 User.you.must.have.privilege=You must have privilege '{0}' in order to assign it.
 User.you.must.have.role=You must have the role '{0}' in order to assign it.
+User.you.must.have.privilege=User you must have the following privileges in order to assign them: {0}
```

Comment:
This is the cause of the error. This line should be `User.you.must.have.privileges` (note the "s" on the end).